packages/code/src/Panel.tsx

```diff
@@ -19,6 +19,18 @@ const copyStyle = css`
   flex-direction: column;
   flex-shrink: 0;
   padding-top: 6px;
+  align-items: center;
+
+  > * {
+    & + * {
+      margin-top: ${spacing[1]}px;
+    }
+  }
```

Comment:
Might be worth simplifying this to `> * + *`.  (There seems to be an empty rule being applied for `> *`)